src/consts/Stages.ts

```diff
@@ -5,6 +5,7 @@ interface STARTSTAGE {
   nextEventTitle: string;
   previous: undefined;
   previousEventTitle: undefined;
+  subStages?: STAGE[];
```

Comment:
This needs to be a ReadonlyArray too
```suggestion
  subStages?: ReadonlyArray<STAGE>;
```